test/setup_bundler_test.rb

```diff
@@ -658,6 +658,62 @@ def test_sets_bundler_version_to_avoid_reloads
     end
   end
 
+  def test_invoke_cli_calls_bundler_directly_for_install
+    Dir.mktmpdir do |dir|
+      Dir.chdir(dir) do
+        File.write(File.join(dir, "gems.rb"), <<~GEMFILE)
+          source "https://rubygems.org"
+          gem "irb"
+        GEMFILE
+
+        Bundler.with_unbundled_env do
+          capture_subprocess_io do
+            system("bundle install")
```

Comment:
Let's pass `exception: true` to the `system` calls to catch any problems early?

Reply:
I'm not opposed to doing that, but we use this same technique in every other tests, so we would need to revisit everything.